tests/integrationv2/common.py

```diff
@@ -54,7 +59,11 @@ def __init__(self, low=8000, high=30000):
 
         # This is a naive way to allocate ports, but it allows us to cut
         # the run time in half without workers colliding.
-        self.ports = iter(range(low + (worker_id * 500), high))
+        worker_offset = (worker_id * chunk_size)
+        base_range = range(low + worker_offset, high)
+        wrap_range = range(low, low + worker_offset)
+        self.ports = iter(itertools.chain(base_range, wrap_range))
```

Comment:
Nit: Shouldn't the naming be swaped:
```
base_range =  range(low, low + worker_offset)
wrap_range =  range(low + worker_offset, high)
```

Reply:
Since we add base_range + wrap_range, that would be equivalent to range(low, high) and all workers would be competing for exactly the same ports.

base_range is the offsetted range the worker starts with. wrap_range is just so that all workers can wrap back around to the beginning and use all ports eventually.